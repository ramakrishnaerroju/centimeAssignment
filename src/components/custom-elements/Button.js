import React from "react";

const Button = ({ callback, btnText }) => {
  return (
    <button className="btn btn-primary" onClick={callback}>
      {btnText}
    </button>
  );
};

export default Button;
