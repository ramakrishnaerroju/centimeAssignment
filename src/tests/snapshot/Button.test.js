import React from "react";
import renderer from "react-test-renderer";
import { Button } from "../../components/custom-elements/index";

test("Button snapshot test", () => {
  const component = renderer.create(<Button />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
