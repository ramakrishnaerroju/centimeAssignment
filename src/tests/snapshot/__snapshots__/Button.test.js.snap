// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot test 1`] = `
<button
  className="btn btn-primary"
/>
`;
